inc/vl53l0x.h

```diff
@@ -24,6 +24,9 @@
  * Public Types
  *****************************************/
 
+/**
+ * @brief Struct to storage the calibration data for one sensor, so the values can be reloaded after reboot
```

Comment:
Acho que aqui fica melhor se quebrar a linha hehe

Reply:
to store né hahaha

Reply:
acontece asuhausuasau